arabic/sleep/sleep.md

```diff
@@ -3,3 +3,6 @@
 لذا ماذا ستفعل الآن؟
 نعم كلامك صحيح  نحن في حاجة إلى العلم ولكن من العوامل الأساسية التي تساعد على تحصيل العلم جيداً هي أخذ قسط
 كافي من النوم لذا سوف أختار أن أنام الآن .  
+كانت رحلة جميلة قضيتها مع زوجتى فى احدى المنتجعات الرائعة وبينما نحن فى طريقنا للعودة ليلا : اذ يالعربة تتعطل فجأة
+ماذا ستفعل؟
+1-[حاول تصليح العربة] (../arabic/fix/try-fix.md) 
```

Comment:
@eyadsalama, you're on the right track! I believe the link `../fix/try-fix.md` would work, and that would also fix the problem where the other page is unreachable.